src/python/scripts/run_gerp.py

```diff
@@ -22,51 +22,86 @@
 
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw
 
-If ``gerpcol`` and/or ``gerpelem`` are not accesible from ``$PATH``, you will need to provide the
+If ``gerpcol`` and/or ``gerpelem`` are not accessible from ``$PATH``, you will need to provide the
 path where they can be found::
 
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
+import argparse
+import json
 import os
 import subprocess
 
-import argschema
 
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    """Enrich the json file with the constrained elements.
 
-class InputSchema(argschema.ArgSchema):
-    """Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment (MSA)."""
+    Args:
+        ce_file: file containing the constrained elements (space-delimited format).
+        json_file: json file with genome coordinate level.
 
-    msa_file = argschema.fields.InputFile(
-        required=True, description="MSA file (MFA format)"
-    )
-    tree_file = argschema.fields.InputFile(
-        required=True, description="Tree file (Newick format). Must include every species in the MSA."
-    )
-    depth_threshold = argschema.fields.Float(
-        required=False,
-        description="Constrained elements' depth threshold for shallow columns, in substitutions "
-                    "per site. By default, 0.5."
-    )
-    gerp_exe_dir = argschema.fields.InputDir(
-        required=False,
-        description="Path where 'gerpcol' and 'gerpelem' binaries can be found. By default, resort to $PATH."
-    )
+    """
+    with open(json_file, "r") as json_file_obj:
+        align_set = json.load(json_file_obj)
 
+    constrained_elems = []
+    with open(ce_file) as ce_file_obj:
+        # enrich json file with constrained elements info
+        for cons_elem in ce_file_obj:
+            if cons_elem.rstrip() == "":
+                continue
+            ce = cons_elem.split()
+            constrained_elem = {
+                "start": int(ce[0]),
+                "end": int(ce[1]),
+                "length": int(ce[2]),
+                "score": float(ce[3]),
+                "p-val": float(ce[4])
+            }
+            constrained_elems.append(constrained_elem)
+    align_set["constrained_elems"] = constrained_elems
 
-if __name__ == "__main__":
-    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
+    with open(json_file, 'w') as json_file_obj:
```

Comment:
An optional post-approval suggestion. Must've missed this last time around:
```suggestion
    with open(json_file, "w") as json_file_obj:
```